filebeat/module/system/_meta/kibana/7/dashboard/Filebeat-ssh-login-attempts.json

```diff
@@ -318,7 +318,7 @@
                         "query": {
                             "query_string": {
                                 "analyze_wildcard": true, 
-                                "query": "_exists_:system.auth.ssh.event"
```

Comment:
I think this one here should be `event.dataset:system.auth and _exists_:event.action`.

It's going to be shaky at best, because we need to improve the event categorization here, to properly identify what's an actual login attempt & so on. Because more event types will eventually get an `event.action`. We're lucky right now only the login attempts populate that.

Reply:
Ok, adjusted. trust you on this one :-)